lib/renderer.js

```diff
@@ -272,6 +272,10 @@
      * Get SVG data for the given component.
      * 
      * @private
+     * @param {Component} component with component.layer or component.comp, or neither.
+     *      If neither is specified, this method gets the SVG data for the whole document.
+     * @param {?Layer} component.layer Layer to get the SVG data for. Exclusive of component.comp.
```

Comment:
Hmm, been looking through the docs and I don't see a way to express "x or y or neither". Generator-core's JSDocs look like the following in this case, which is what I did:
```
* @param {number=} settings.compId Layer comp ID (exclusive of settings.compIndex)
* @param {number=} settings.compIndex Layer comp index (exclusive of settings.compId)
```

I also realized I should use "=" instead of "?" for optional parameters and "!" for non-nullable parameters. I'll update my docs to do that.